__tests__/RouterLink.spec.ts

```diff
@@ -792,7 +795,7 @@ describe('RouterLink', () => {
       { to: locations.basic.string },
       locations.basic.normalized
     )
-    wrapper.find('a')!.click()
+    wrapper.find('a')!.trigger('click')
```

Comment:
click should still work 🤔 
We can (and should) remove the `await nextTick()` and `await wrapper.find(...).click()`

Reply:
`.click()` is not work any more.